app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -330,14 +334,17 @@ sealed class DaxFireDialogCta(
 
     override fun pixelOkParameters(): Map<String, String> = emptyMap()
 
-    override fun pixelShownParameters(): Map<String, String> = emptyMap()
+    override fun pixelShownParameters(): Map<String, String> = mapOf(Pixel.PixelParameter.CTA_SHOWN to addCtaToHistory(ctaPixelParam))
 
-    class TryClearDataCta : DaxFireDialogCta(
+    class TryClearDataCta(override val onboardingStore: OnboardingStore, override val appInstallStore: AppInstallStore) : DaxFireDialogCta(
```

Comment:
Not a big deal but you could test to check the `ctaPixelParam`, etc have the expected values. We do have a few simple tests like that for other CTAs.